lib/rage/setup.rb

```diff
@@ -8,6 +8,8 @@
 # Load application classes
 Rage.code_loader.setup
 
+Rage.__router.set_params_wrappers
```

Comment:
Initiate parameters wrappers setting after controller classes are loaded and flags that control wrappers plugging in are set.